src/components/primitives/Links/ExternalLink.tsx

```diff
@@ -0,0 +1,30 @@
+import { ParamDetail } from 'components/ActionsBuilder/CallDetails/CallDetails.styled';
+import { BiLinkExternal } from 'react-icons/bi';
+import { FiExternalLink } from 'react-icons/fi';
+import { StyledSegmentLink } from './styles';
+import { ExternalLinkProps } from './types';
+import { UnstyledExternalLink } from './styles';
+
+export const ExternalLink: React.FC<ExternalLinkProps> = ({
+  href,
+  children,
+  unstyled,
+}) => {
+  if (unstyled) {
+    return (
+      <UnstyledExternalLink href={href} target="_blank" rel="noopener">
+        <ParamDetail>

```

Comment:
The styled and unstyled links place the icon differently, caused by the `<ParamDetail />` component.

I think the behaviour in the unstyled component is the desired one (the icon besides the text), but I'd add some spacing between the elements.

![image](https://user-images.githubusercontent.com/75996796/200829796-ae7b6b11-4e1f-4050-90af-37a2ef85ff8d.png)